lib/updateIndOk.ml

```diff
@@ -930,6 +930,8 @@ let effective_del_no_commit base op =
 
 let effective_del_commit conf base op =
   Notes.update_notes_links_db base (Def.NLDB.PgInd op.key_index) "";
+  let key = (Name.lower op.first_name, Name.lower op.surname, op.occ) in
```

Comment:
Factorize

Reply:
There are conversion functions between person and `gen_person` in Gwdb_driver. Maybe event have a key building function for each ?